src/plugins/data/common/field_formats/converters/duration.ts

```diff
@@ -89,59 +90,89 @@ const inputFormats = [
   },
 ];
 const DEFAULT_OUTPUT_FORMAT = {
-  text: i18n.translate('data.fieldFormats.duration.outputFormats.humanize', {
-    defaultMessage: 'Human Readable',
+  text: i18n.translate('data.fieldFormats.duration.outputFormats.humanize.precise', {
+    defaultMessage: 'Human Readable precise',
```

Comment:
Nit copy: I think casing should be consistent here. It should either be `Human Readable Precise` or `Human readable precise`. 

As I typing it out autocorrect actually suggesting me that this needs a hyphen. Then how about: 

`Human-readable (precise)` and `Human-readable (approximate)` ?

Reply:
I liked the suggestions. Done